pysap/plugins/mri/reconstruct/utils.py

```diff
@@ -61,6 +61,30 @@ def convert_locations_to_mask(samples_locations, img_shape):
     return mask
 
 
+def normalize_samples(samples):
```

Comment:
This function has a very generic name, but a very specific purpose. For example, the [-0.5; 0.5[ range is hard-wired.

I suggest that you either 
a) update the name *e.g.* `normalize_non_cartesian_samples`, in which case you should remove it from numerics and reference it directly from the plugin, or
b) you update the function such that you can specify the desired range (you can keep [-0.5; 0.5[ as the default) and the second dimension of the input samples is flexible.

Reply:
This range is a convention for appropriately all the packages that are implementing non-Cartesian Fourier transform. 
Even the PR form that implements the NUFFT operations is based on it, I agree with you it should be renamed as `normalize_non_cartesian_samples`.
This is basically the first step that needs to be applied for non-cartesian reconstruction just to make sure that the non-cartesian fourier operator will be set correctly.
So maybe it has to be put in the `numerics/fourier.py` or what could be done is a check in the in the `__init__` of `NFFT` operator before the instantiation.